controller/auth_utility_test.go

```diff
@@ -22,13 +22,14 @@ import (
 
 const (
 	// the various HTTP endpoints
+	endpointWorkItem               = "/api/workitems"
```

Comment:
I think I miss a lot of context. So we revert the changes to have a space for the workitems, do we ?